packages/destination-actions/src/destinations/heap/trackEvent/index.ts

```diff
@@ -96,31 +98,36 @@ const action: ActionDefinition<Settings, Payload> = {
       throw new IntegrationError('Either anonymous user id or identity should be specified.')
     }
 
-    const defaultEventProperties = { segment_library: payload.library_name || HEAP_SEGMENT_LIBRARY_NAME }
+    const defaultEventProperties = { segment_library: HEAP_SEGMENT_LIBRARY_NAME }
```

Comment:
What's the reason for removing `payload.library_name`?

In cloud mode, Segment provides a value for `library_name`, for example, `segment-node` for server-side Node SDK. It makes sense to provide a default value because Heap internals require events to have `segment_library` property, but it's unclear to me what's the reason for removing the provided value.

Reply:
We want to know where the events are coming from. Unless we add another property, this is the only way that we know. The plan is to add a metric that will tell us how many calls and customers are using the cloud mode integration and the browser mode integration. Do we have much use for the value segment will provide?

Reply:
That makes sense. We don't use this value for internal purposes, but this value exists in the existing cloud mode integration. So there's a small chance that Heap customers use this. I don't have a strong opinion here, so it's up to you.